main.go

```diff
@@ -123,17 +142,131 @@ func NewExporter(server string, timeout time.Duration) *Exporter {
 			nil,
 		),
 		evictions: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "", "items_evicted_total"),
+			prometheus.BuildFQName(namespace, "", "evicted_total"),
 			"Number of valid items removed from cache to free memory for new items.",
 			nil,
 			nil,
 		),
 		reclaimed: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "", "items_reclaimed_total"),
+			prometheus.BuildFQName(namespace, "", "reclaimed_total"),
 			"Number of times an entry was stored using memory from an expired entry.",
 			nil,
 			nil,
 		),
+		active_slabs: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "active_slabs_total"),
+			"Total number of slab classes allocated.",
+			nil,
+			nil,
+		),
+		malloced: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "malloced_bytes"),
+			"Number of bytes of memory allocated to slab pages.",
+			nil,
+			nil,
+		),
+		itemsNumber: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "items_number_total"),
```

Comment:
Instead of renaming these metrics, I'd use a common "subsystem" for all the metrics coming from the slab+items statistics commands, probably called `slab` or the plural `slabs`. So all metrics with a slab label would have the common prefix `memcached_slab_*`. Memcached uses the term `number` for the slab identifier, so I'd use that as label name instead of `slab`.

This applies to all slab related metrics.